features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -111,6 +111,17 @@
   step %{I open signups for "#{title}"}
 end
 
+Given /^the open gift exchange "([^\"]*)" has overlapping tags$/ do |challengename|
```

Comment:
When possible, we are moving towards [Cucumber Expressions](https://github.com/cucumber/cucumber-expressions#readme). I think the following would work
```suggestion
Given "the open gift exchange {string} with overlapping tags" do |challenge_name|
```
I've found https://cucumber.github.io/try-cucumber-expressions helpful to play around with the syntax